lib/shared/config/loadfiles.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var get = require('lodash.get');
+var set = require('lodash.set');
+var isString = require('lodash.isstring');
+var configSpecs = require('./cfg-specs');
+
+function loadConfigFiles(configFilesBase, keysInOrder) {
+  var config = {};
+
+  var configFilePaths = keysInOrder.map(function(key) {
```

Comment:
Does `fined` not do this for us automatically?  It's okay if it doesn't but my mind is fuzzy on it.